src/transformers/pipelines.py

```diff
@@ -1299,6 +1299,29 @@ def __call__(self, *args, targets=None, **kwargs):
         return results
 
 
+class TokenClassificationArgumentHandler(ArgumentHandler):
```

Comment:
added this to check `offset_mapping` if provided. (does a simple batch_size check)